package/kernel/linux/modules/video.mk

```diff
@@ -1142,7 +1142,7 @@ $(eval $(call KernelPackage,video-dma))
 
 define KernelPackage/video-coda
   TITLE:=i.MX VPU support
-  DEPENDS:=@(TARGET_imx&&!TARGET_imx_cortexa7) +kmod-video-mem2mem +kmod-video-dma
+  DEPENDS:=@(TARGET_imx&&TARGET_imx_cortexa9) +kmod-video-mem2mem +kmod-video-dma
```

Comment:
Why this switch to cortexa9 here? This probably needs to go to a separate commit

Reply:
I just inverted the logic because the modules are valid for coretexa9 but not valid for cortexa7 and cortexa53 (or at least not yet valid for cortexa53)

Reply:
in the case of cortexa53, or at least imx8mm the vpu is hantro and not coda

Reply:
Okay, I missed that negation.